pyiode/objects/equation.pyx

```diff
@@ -148,6 +153,9 @@ cdef class Equation:
             raise TypeError("'name': Expected value of type string.\nGot value of type '" + type(name).__name__ + "'")
         self.c_equation.set_lec(lec.encode(), name.encode())
 
+    # GDM> I think the property is enough. If you want to make setting only one bound easier,
```

Comment:
Yes, the property is enough. I removed the set_sample() method and added examples in doctstring of the sample property -> PR #580